data/nerdynikhil.json

```diff
@@ -2,6 +2,8 @@
     "name": "Nikhil Barik",
     "type": "personal",
     "bio": "Software Engineer at Yahoo. Loves FOSS.",
+    "displayStatePublic": true,
+    "avatar": "http://github.com/nerdynikhil.png",
```

Comment:
Hi @nerdynikhil, you can remove these 2 lines, these are not required. Thank you :+1:

Reply:
```suggestion
```